tests/acceptance-tests/test/specs/smoke-tests/user-onboarding.app-test.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+require("dotenv").load();
+const yaml = require("js-yaml");
+const fs   = require("fs");
+const expect = require("chai").expect;
+const getId = require("../../../lib/get-elements.js");
+
+let baseUrl;
+beforeEach(function () {
+  const browserConfig = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/settings.yml", "utf8"));
+  baseUrl = process.env.REACTION_BASE_URL || browserConfig.base_url.toString();
+  browser.url(baseUrl);
+  // browser.getSession().then(function (sessionid) {
```

Comment:
Remove dead code.

Reply:
Ok @mradeybee I will remove it but that was from reaction commerce not our comment